python-sdk/src/langwatch/prompt/prompt.py

```
@@ -0,0 +1,72 @@
+from .formatter import PromptFormatter
+import langwatch
+from typing import List, Any, Dict
+
+class Prompt:
+    """
+    A class representing a prompt configuration that can be used with OpenAI's API.
+    Handles formatting messages with variables.
+    """
+    def __init__(self, config: Any, formatter: PromptFormatter = PromptFormatter()):
```

Ruff (0.11.9)

Do not perform function call `PromptFormatter` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)